packages/styles/forms.css

```
@@ -11,6 +11,7 @@
   --field-border-color-error-hover: #c42f41;
   --field-border-color-error-focus-glow: rgba(217, 50, 81, 0.7);
   --field-border-color-disabled: rgba(204, 204, 204, 0.25);
+  --field-border-radius: var(--space-half);
```

This is inappropriate usage of `--space-half`. Spacing variables are intended to be used for spacing things, like padding and margins. We can just outright set border radius to 4px.

```suggestion
  --field-border-radius: 4px;
```